boot/boot.js

```diff
@@ -1139,13 +1139,13 @@ $tw.Wiki.prototype.deserializeTiddlers = function(type,text,srcFields) {
 		fields = Object.create(null);
 	if(!deserializer && $tw.config.fileExtensionInfo[type]) {
 		// If we didn't find the serializer, try converting it from an extension to a content type
-		type = $tw.config.fileExtensionInfo[type].type;
```

Comment:
Hi @danielo515 sorry for the late response. I'm confused because this change changes the type of the variable `type`, which is still used further down this same function as though it were a string (see `return deserializer.call(this,text,fields,type);`).